packages/form-js-editor/src/render/components/Util.js

```diff
@@ -8,4 +8,105 @@ export function editorFormFieldClasses(type, { disabled = false } = {}) {
   return classNames('fjs-form-field', `fjs-form-field-${type}`, {
     'fjs-disabled': disabled
   });
+}
+
+/**
+ * Add a dragger that calls back the passed function with
+ * { event, delta } on drag.
+ *
+ * @example
+ *
+ * function dragMove(event, delta) {
+ *   // we are dragging (!!)
+ * }
+ *
+ * domElement.addEventListener('dragstart', dragger(dragMove));
+ *
+ * @param {Function} fn
+ * @param {Element} dragPreview
+ *
+ * @return {Function} drag start callback function
+ */
+export function createDragger(fn, dragPreview) {
```

Comment:
For reference: I had a look at some existing resizable component solutions (e.g., https://github.com/react-grid-layout/react-resizable, https://www.npmjs.com/package/preact-moveable), but none of them gave enough flexibility to achieve what we wanted; especially due to the fact we resize in columns, not pixels.

I decided to re-use proven utilities from bpmn-io to implement the behaviors.